README.md

```diff
@@ -33,13 +33,13 @@ Check out [Matisse releases](https://github.com/zhihu/Matisse/releases) to see m
 If you use [Glide](https://github.com/bumptech/glide) as your image engine, add rules as Glide's README says.  
 And add extra rule:
 ```pro
--dontwarn com.squareup.picasso.**
+-dontwarn com.bumptech.glide.**
 ```
 
 If you use [Picasso](https://github.com/square/picasso) as your image engine, add rules as Picasso's README says.  
 And add extra rule:
 ```pro
--dontwarn com.bumptech.glide.**
+-dontwarn com.squareup.picasso.**
```

Comment:
You get it wrong. See https://github.com/zhihu/Matisse/pull/102.

Reply:
Yup, I got that. thanx